### DISCUSSION TITLE
keyword change in "Importing modules" section

### DISCUSSION THREAD

**Discussion by mohammadpasha** [pybind/pybind11#3714] [General]
on https://pybind11.readthedocs.io/en/stable/advanced/embedding.html which explains importing modules needs a correction.
_py::module__  should be  _py::module_ . It gave compile time errors. 

Compiler output:
**error: no member named 'module_' in namespace 'pybind11'; did you mean 'module'?
    py::module_ sys = py::module_::import("sys");
                      ~~~~^~~~~~~
                          module
/usr/include/pybind11/pybind11.h:783:7: note: 'module' declared here
class module : public object {**

**Comment by henryiii** [comment]
I think you have an old version of pybind11. We renamed `module` to `module_` to avoid issues with C++20's soft keyword quite a while back, maybe 2.7 or so.
